packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -156,16 +156,20 @@ class LongPressGestureRecognizer extends PrimaryPointerGestureRecognizer {
   /// subsequent callbacks ([onLongPressMoveUpdate], [onLongPressUp],
   /// [onLongPressEnd]) will stop. Defaults to null, which means the gesture
   /// can be moved without limit once the long press is accepted.
+  ///
+  /// The [duration] argument can be used to overwrite the default duration
+  /// after which the long press will be recognized.
   LongPressGestureRecognizer({
+    Duration duration,
```

Comment:
I originally called it deadline, but it seems a bit off to expose the inner workings by using that name. So I called it duration instead since it will be clear to anyone using it what it means.

Reply:
Got it, that makes sense.